.github/workflows/main.yaml

```diff
@@ -132,6 +132,11 @@ jobs:
 
   e2e_test:
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        ENABLE_QUIC:
+          - false
+          - true
```

Comment:
```suggestion
        protocol:
          - websocket
          - quic
```

Reply:
talked with @fisherxu ,  let us keep this